src/ontology/mondo-ingest.Makefile

```diff
@@ -169,6 +169,28 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# TODO: Add .PHONY here and below to top of file?
+# TODO: need a purl for this?
+# TODO: use this target: tmp/mondo.sssom.tsv
+.PHONY: mondo-mappings
```

Comment:
no .PHONY goals in the middle of pipelines..

Reply:
Yep! I remember our discussion. These changes are on the way (actually these targets will be removed anyway)

Reply:
Yep, I remember bringing this up yesterday. No worries, will be done.

Reply:
done